codelists/sustainabilityGoal.csv

```diff
@@ -1,15 +1,30 @@
 Code,Title,Description
 economic,Economic,"The goal is economic, such as formal employment and the local economy."
 economic.formalEmploymentPromotion,Formal employment promotion,The goal is to promote formal employment.
+economic.innovativePurchase,Innovative purchase,"The goal is to procure innovative goods, services or works."
 economic.localEconomyPromotion,Local economy promotion,The goal is to promote the local economy.
 economic.marketInnovationPromotion,Market innovation promotion,The goal is to promote market innovation.
+economic.processInnovation,Process innovation,"The goal is to procure goods, services or works that entail process innovation."
+economic.productInnovation,Product innocation,"The goal is to procure goods, services or works that entail product innovation."
+economic.researchDevelopmentActivities,Research and development activities,"The goal is to procure goods, services or works that involve research and development activities."
```

Comment:
If we decide to remove Promotion from existing codes, then we can remove Activities here.

Reply:
I committed the suggestion to add Promotion so I'll resolve this comment.